app/services/katello/registration_manager.rb

```diff
@@ -128,6 +128,7 @@ def joined_hostnames(hosts)
       def unregister_host(host, options = {})
         organization_destroy = options.fetch(:organization_destroy, false)
         unregistering = options.fetch(:unregistering, false)
+        reregistering = options.fetch(:reregistering, false)
```

Comment:
Is `reregistering` an accurate name? Has the host been registered before at this point?

Reply:
I suppose I could change it to something like `keep_kickstart_repo`. 

I named it `reregistering` originally because it communicates that the unregister code is running before being expected to register again later.